src/components/subscriptions/SubscriptionExpirationModal.test.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { Router, Route } from 'react-router-dom';
+import { createMemoryHistory } from 'history';
+import { Modal } from '@edx/paragon';
+
+import { MODAL_DIALOG_CLASS_NAME, BaseSubscriptionExpirationModal } from './SubscriptionExpirationModal';
+import { SubscriptionContext } from './SubscriptionData';
+import { SUBSCRIPTION_DAYS_REMAINING_MODERATE } from './constants';
+import { addDays, getSubscriptionExpirationDetails } from './test-utils';
+
+
+const history = createMemoryHistory({
+  initialEntries: ['/'],
+});
+
+jest.mock('universal-cookie', () => {
+  const mockCookie = {
+    get: jest.fn(() => true),
+  };
+  return jest.fn(() => mockCookie);
+});
```

Comment:
Mocking this cookie stuff for separate tests is not going to well for me. I can do what I have right now and get it to always be true or false, but I'm having trouble clearing the mocks so that both the cookie and non-cookie test work.
What I've tried with `doMock`, `setMockImplementation`, `jest.spyOn` hasn't worked, so please let me know if you all have other ideas

Reply:
Did you try not mocking it? If it's just a wrapper class around standard front-end cookies (i.e. document.cookie) then you might not need to mock it at all. And you could try manually clearing the cookies after each test as you would normally using the Cookie class.

Reply:
I have tried that, that gives the opposite issue. I have a different idea I'm trying however